website/versioned_docs/version-7.0.0/plugin-syntax-do-expressions.md

```diff
@@ -5,6 +5,10 @@ sidebar_label: syntax-do-expressions
 original_id: babel-plugin-syntax-do-expressions
 ---
 
+> #### Syntax only
+>
+> It's unlikely you want to use this plugin directly as it only enables Babel to parse this syntax. Instead, use [plugin-proposal-do-expressions](babeljs.io/docs/en/plugin-proposal-do-expressions.md) to _both_ parse and transform this syntax.
```

Comment:
These should be like:

```
[plugin-proposal-do-expressions](plugin-proposal-do-expressions.md)
```

You can checkout https://raw.githubusercontent.com/babel/website/master/website/versioned_docs/version-7.0.0/plugins.md for more examples.

We want these to be relative to the version you're viewing.

Reply:
@doms no worries, no lie... it's not really that intuitive :P